swagger.yaml

```diff
@@ -557,7 +557,7 @@ paths:
           type: integer
         - name: resource
           in: query
-          description: Resource id, for filtering reservations by resource
+          description: Resource id, for filtering reservations by resource. Accepts multiple comma-separated values.
```

Comment:
Does this also work by providing multiples of the same query parameter?
As in `localhost:8000/v1/reservation?resource=abc&resource=123`

Reply:
Pretty sure it does not, should it? I don't think other filters in Respa do that either (?)